lib/schema.js

```diff
@@ -495,7 +499,17 @@ Schema.prototype.pick = function(paths, options) {
   }
 
   for (const path of paths) {
-    if (this.nested[path]) {
+    if (path in this.encryptedFields) {
```

Comment:
Is this code path necessary? `newSchema.add({ [path]: schematype });` ideally should handle adding the full schematype to the new schema correctly. Is the issue that `this.encryptedFields` isn't copied over?

Reply:
Yeah, because schematype doesn't include encryption information.

Its been awhile but I think I intentionally decided against storing encryption information on the SchemaType because it seemed like a separate concern.  SchemaTypes declare the type of the field, encryption configuration is declared on a schema.

I could reconsider though, if you think that is preferable.